bundles/NewsletterBundle/doc/19_Newsletter.md

```diff
@@ -73,8 +68,8 @@ For example, `\NewsletterController`: `src/Controllers/NewsletterController.php`
 
 namespace App\Controller; 
 
+use Pimcore\Bundle\NewsletterBundle\Tool\Newsletter;
 use Pimcore\Model;
```

Comment:
Here also need a change in the image url path prefix
`![Create email document](../img/newsletter_create_document_confirmation.png)`